src/main/java/seedu/agendum/logic/commands/StoreCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.agendum.logic.commands;
+
+import seedu.agendum.commons.core.Config;
+import seedu.agendum.commons.util.FileUtil;
+import seedu.agendum.commons.util.StringUtil;
+
+/**
+ * Allow the user to specify a folder as the data storage location
+ */
+public class StoreCommand extends Command {
+    
+    public static final String COMMAND_WORD = "store";
+    public static final String MESSAGE_SUCCESS = "New save location: %1$s";
+    public static final String MESSAGE_LOCATION_INVALID = "The specified location is invalid.";
+    public static final String MESSAGE_LOCATION_DEFAULT = "Save location set to default: %1$s";
+    
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Specify a save location. \n"
+            + "Parameters: FILE_PATH\n" 
+            + "Example: " + COMMAND_WORD 
+            + "C:/agendum";
```

Comment:
Inconsistent as you are getting them to specify the file name too. Decide if file name is needed and update comments/ the user guides
- [ ] Fixed